specification/columns/listunitprice.md

```diff
@@ -16,6 +16,10 @@ List Unit Price
 
 The suggested provider-published unit price for a single Pricing Unit of the associated SKU, exclusive of any discounts.
 
+## Usability Constraints
+
+**Aggregation:** This column does not support aggregation.
```

Comment:
This feels incomplete to me.  I think something _like_ the suggestion below is slightly better.  Definitely open to other ideas than the suggestion too.

```suggestion
**Aggregation:** Column values should only be viewed in context of its row and not aggregated to produce a total.
```

Reply:
agreed